File: cpp/tests/exams/exams1/test_Exam3_2.cpp
#include "gtest/gtest.h"

#include "exams1.h"

TEST(Exam3_2, test1)
{
    ASSERT_EQ("III", intToRoman(3));
    ASSERT_EQ("IV", intToRoman(4));
    ASSERT_EQ("IX", intToRoman(9));
    ASSERT_EQ("LVIII", intToRoman(58));
}

File: python/tests/exams/exams1/test_Exam3_2.py
from unittest import TestCase

from exams.exams1.Exam3_2 import int_to_roman


class TestExam(TestCase):

    def test_int_to_roman(self):
        self.assertEqual('III', int_to_roman(3))
        self.assertEqual('IV', int_to_roman(4))
        self.assertEqual('IX', int_to_roman(9))
        self.assertEqual('LVIII', int_to_roman(58))
        self.assertEqual('MCMXCIV', int_to_roman(1994))
